wiki/howToUseSemantics.md

```diff
@@ -0,0 +1,42 @@
+## How to use semantics
+
+To import the project user need to add to pom.xml
```

Comment:
To import the project, in pom.xml add the below dependency/dependencies

Reply:
changed as suggested